### PR TITLE
Add 5 minute picker #3

### PR DESCRIPTION
**Pull request by lkristensen**
When creating a new event, this contribution will change the time picker to present data in 5 minute intervals instead of 1 minute intervals.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,4 @@ debian/*.substvars
 builddir/
 Testing/
 build-*
+armhf-build/
```
Comment (NeoTheThird) [inline .gitignore]: Redundant, since there's a wildcard option above

**Comment by NeoTheThird** [comment]
@lkristensen Are you also on telegram? That way we could communicate a little more efficiently.

**Comment by NeoTheThird** [comment]
I added you as a contributor on this app, so you can also work on a branch in this repository, if you want. Let's keep branch protection on master for now, that way we can work on different branches and then merge the changes using reviewed pull requests. Unfortunately, we have no guidelines for this in place yet, but this workflow would allow us to set up CI and automatic publication of the master branch to the OpenStore in the future, which would be very handy.

From what i can see, your changes are solid. Do you want me to upload it into the OpenStore? I'll also make sure it's pre-installed in future images, if you say it's ready for release.

**Comment by lkristensen** [comment]
I just contacted you on Telegram. The changes should be in working order, so please publish them.

**Comment by NeoTheThird** [comment]
I added you as a contributor on this app, so you can also work on a branch in this repository, if you want. Let's keep branch protection on master for now, that way we can work on different branches and then merge the changes using reviewed pull requests. Unfortunately, we have no guidelines for this in place yet, but this workflow would allow us to set up CI and automatic publication of the master branch to the OpenStore in the future, which would be very handy.

From what i can see, your changes are solid. Do you want me to upload it into the OpenStore? I'll also make sure it's pre-installed in future images, if you say it's ready for release.

**Pull request merged**